lib/src/data/datasource/remote/pokedex_api_service.dart

```diff
@@ -20,11 +19,15 @@ class PokedexApiService {
         final pokedexModel = PokedexModel.fromJson(
           json.decode(response.body),
         );
-        await Future.forEach<PokemonEntryModel>(
-            pokedexModel.pokemonEntries as List<PokemonEntryModel>,
-            (pokemonEntry) async {
-          pokemonEntry.pokemon = await _getPokemon(pokemonEntry.entryNumber);
-        });
+        for (int i = Constants.initialValue;
+            i < pokedexModel.pokemonEntries.length;
+            i++) {
+          pokedexModel.pokemonEntries[i] =
+              pokedexModel.pokemonEntries[i].copyWith(
+            pokemon:
+                await _getPokemon(pokedexModel.pokemonEntries[i].entryNumber),
+          );
+        }
```

Comment:
This is a kinda weird format

Reply:
It looks like this when I do control + alt + l